classes/Tools.php

```diff
@@ -2344,6 +2345,11 @@ public static function getBrightness($hex)
         return (($r * 299) + ($g * 587) + ($b * 114)) / 1000;
     }
 
+    public static function isBright($hex)
+    {
+        return (new ColorBrightnessCalculator())->isBright($hex);
```

Comment:
Could you add a way to don't reinstanciate `ColorBrightnessCalculator` each time this function is called please :)